### PR TITLE
Feature/8992 improve backdoor signatures

### LINKED ISSUE #8992: Improve signatures to detect backdoors
## History
I refactored the find_backdoors plugin in order to use specific signatures, which might get outdated.

## Task
 - [ ] Find new signatures to detect web shells and backdoors
 - [ ] Update the signatures file using the shells at https://github.com/tennc/webshell
 - [ ] Update the signatures file using https://github.com/Neo23x0/Loki/blob/master/signatures/thor-webshells.yar , note that some signatures match source code and we don't have access to it, just to the rendered response
 - [ ] Create a task just like this one so I remember to keep doing this periodically

### PR DESCRIPTION
**Pull request by Q-back**
Grepped filenames and signatures from https://github.com/tennc/webshell. Closes #8992

### CODE REVIEW

File: w3af/plugins/crawl/find_backdoors/web_shells.txt [hunk 1]
```diff
@@ -178,3 +178,721 @@ nc.exe
 netcat.exe
 socat.exe
 cmd.pl
+
```
Comment (andresriancho) [inline w3af/plugins/crawl/find_backdoors/web_shells.txt]: Not 100% sure about this, but how are empty lines in this file handled by the plugin? Same with the previous file.

**Comment by andresriancho** [comment]
Please confirm that the empty line is not an issue and we'll merge this!

File: w3af/plugins/crawl/find_backdoors/web_shells.txt (hunk 1 above)
Comment (Q-back) [reply w3af/plugins/crawl/find_backdoors/web_shells.txt]: Thanks to `line.strip()` empty lines are ignored. 
https://github.com/andresriancho/w3af/blob/193738bf46dd4d9476e6f632df4204ddf71227fb/w3af/plugins/crawl/find_backdoors.py#L103
```
if not line:
    continue
```
https://github.com/andresriancho/w3af/blob/193738bf46dd4d9476e6f632df4204ddf71227fb/w3af/plugins/crawl/find_backdoors.py#L108

**Comment by andresriancho** [comment]
giphy

**Pull request merged**